packit/cli/propose_downstream.py

```diff
@@ -115,6 +117,14 @@ def sync_release_common_options(func):
         is_flag=True,
         help="Don't discard changes in the git repo by default, unless this is set.",
     )
+    @click.option(
+        "-b",
+        "--resolve-bug",
+        help="Bug(s) that are resolved with the update, e.g. rhbz#123 (multiple can be specified)",
```

Comment:
I was wondering if they should be comma separated or just space separated, I think space but I am not sure... Maybe we can add this to the description?

Reply:
There is `multiple=True` which means you can do `-b rhbz#123 -b rhbz#321`.